_plugins/eiti-format.rb

```diff
@@ -0,0 +1,71 @@
+require 'liquid'
+
+module EITI
+
+  module Format
+
+    # returns the singular suffix of a value if the value is 1,
+    # otherwise the singular form
+    #
+    # >> EITI::Format.plural(100)
+    # => 's'
+    # >> EITI::Format.plural(1)
+    # => ''
+    # >> EITI::Format.plural(1, 'ies', 'y')
+    # => 'y'
+    # >> EITI::Format.plural(2, 'ies', 'y')
+    # => 'ies'
+    def plural(num, plural = 's', singular = '')
+      num.to_i == 1 ? singular : plural
+    end
+    module_function :plural
```

Comment:
These can/should be refactored to the top of the file. If you add me to the project, I'll push a commit on to this PR